I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml

```diff
@@ -1486,11 +1499,11 @@ Line |     .  +-----+ | |  *  *  | | +-----+  .     | Line
    <xref target="sect-2.3.4"/>. These MCs can be established between two
    line ports of the ROADM or between a line port and an Add/Drop port of
    the ROADM. The Add/Drop ports of a ROADM are those ports to which optical
-   transponders are connected. Typically, this is a single channel signal
-   (single OTSi), but principally this could also be a group of OTSi
-   signals. The optical impairments associated with these MCs are different
-   and the paths of the MCs inside the ROADM node can be categorized as
-   follows:
+   transponders are connected. Typically, Add/Drop ports deal with a single 
```

Comment:
Editorial: replace Add/Drop with add/drop (to be consistent with the rest of the document).

Reply:
done